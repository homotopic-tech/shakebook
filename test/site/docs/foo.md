---
title: The Foo In The Valley
description: ruff
---

Lorem ipsum
