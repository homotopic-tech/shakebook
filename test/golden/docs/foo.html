<html>
  <head>
    <title>The Foo In The Valley</title>
  </head>
  <body>
    <h1>This post page is called The Foo In The Valley</h1>
    <p>It was generated from the source test/site/docs/foo.md</p>
    <p>For the url </p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the recent posts again:
      <ul>
        <li>
          <a href="/docs/bar.html">The Bar in the mountains</a>
          <p>Hi im champ</p>
        </li>
        <li>
          <a href="/docs/baz.html">The Baz In the Hills</a>
          <p>woof</p>
        </li>
      </ul>
    </p>
  </body>
</html>
